srcs/server/serverChild.cpp

```diff
@@ -222,11 +232,13 @@ namespace ft
         plane_uri = get_uri_and_check_CGI(path_, tmp_query, tmp_is_cgi);
 
         if (stat(plane_uri.c_str(), &sb) == -1) {
-			std::cout << "URIII not good\n";
+			std::cout << "ERNO: " << errno << std::endl;
 			if (errno == EACCES)
 				throw_(403, "forbidden - cannot access uri path");
-			else if (errno == ENOENT)
-				throw_(404, "forbidden - path doesn't exist");
+			else if (errno == ENOENT) {
+				if (needExistingFile)// PUT & POST can be ENOENT
+					throw_(404, "NotFound - path doesn't exist");
+			}
```

Comment:
Is this true? or only PUT can be ENOENT? (file doesn't exist).

Reply:
I think it's probably okay